content/en/news/releases/1.20.x/announcing-1.20.1/index.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Announcing Istio 1.20.1
+linktitle: 1.20.1
+subtitle: Patch Release
+description: Istio 1.20.1 patch release.
+publishdate: 2023-12-12
+release: 1.20.1
+---
+
```

Comment:
```suggestion

This release implements the security updates described in our Dec 11th post, [`ISTIO-SECURITY-2023-005`](/news/security/istio-security-2023-005) along with bug fixes to improve robustness.

```

Reply:
Should that have today's date, since the post is likely to merge today?

Reply:
The date on the security PR still has 12/11 in it. We can update the date in the security PR and then update these lines in the various PRs to reflect it as well.

Reply:
Should I just commit the suggestion?